modules/account-lib/test/unit/coin/algo/transactionBuilder/transferBuilder.ts

```diff
@@ -11,6 +11,7 @@ describe('Algo Transfer Builder', () => {
 
   const sender = AlgoResources.accounts.account1;
   const receiver = AlgoResources.accounts.account2;
+  const ALGOTXNLENGTH = 52;
```

Comment:
why 52?

Reply:
This is used in other tests that are not mine. But I've changed the test we saw earlier that was using this.